package.json

```diff
@@ -19,13 +19,13 @@
     "@changesets/cli": "^2.25.0",
```

Comment:
can we add a `start` script in the main `package.json`:

```

    "start:test-appt": "yarn workspace @gnosis.pm/safe-apps-test-app start",
```

And update the `Readme.md` file

Reply:
done senor